app/services/invoices/refresh_draft_and_finalize_service.rb

```diff
@@ -12,8 +12,12 @@ def call
       return result unless invoice.draft?
```

Comment:
Can you please also update `FinalizeJob` and `RefreshDraftJob` so that we don't generate dead job for each tax error

Reply:
🥇